src/main/resources/templates/java/test/gradle/projectTemplate/build.gradle

```
@@ -1,11 +1,15 @@
+// %static-code-analysis-start%
 plugins {
-    id 'java'
-    // %static-code-analysis-start%
     id 'checkstyle'
     id 'pmd'
     id 'com.github.spotbugs' version '5.0.6'
-    // %static-code-analysis-stop%
 }
+// %static-code-analysis-stop%
+
+apply plugin: 'java'
+sourceCompatibility = 17
+version = '1.0.0'
+compileJava.options.encoding = 'UTF-8'
```

Otherwise I get an Ares-error when executing tests on windows (`Warnung: Standard-Charset ist nicht UTF-8: windows-1252 at de.tum.in.test.api.io.IOTester.checkEncoding(IOTester.java:227)` )
```suggestion
compileJava.options.encoding = 'UTF-8'
compileTestJava.options.encoding = "UTF-8"
```

---

You probably want this to be `'UTF-8'` instead of `"UTF-8"` to be consistent